code/stable/glassbr/src/cpp/README.md

```diff
@@ -1,7 +1,22 @@
-###Summary of Key Information for Running GlassBR on C++
-C++ Version: gcc 10.1
-
+# GlassBR 
+> Authors:  Nikitha Krithnan, W. Spencer Smith 
+ >  __purpose__
```

Comment:
Is this right?  Should `__purpose__` be in the document like that?

Reply:
No the __purpose__ is not supposed to be there, it is a place holder because this issue is incomplete.

According to Dr Smith we should have the purpose in the Readme's, however the purpose is stored as a sentence. In order to convert sentences to a doc I need several more parameters such as ChunkDB, Stage and Linearity (there is a function in drasil-printers sentenceToDoc that needs these). However due to the discussion of whether or not "System Information" should exist, I held out on such as large change.

Reply:
Thanks. So I guess the question (@smiths ) is whether to merge this in now anyways, or wait.